### ISSUE TITLE
do you continue the project?

### ISSUE THREAD

**Issue by onoff0** [unwashed-and-dazed/visallo#1]
are you going to continue to improve the project?

**Comment by unwashed-and-dazed** [comment]
well, i was planning to install it to test it, but I lack java skills. When I realized they had closed the source code I uploaded the version i had cloned. Do you have any information to help me getting this started? is it broken?

**Comment by lorenzoromani1983** [comment]
any news about this? it gives me mojofailure error (webpack.js) and can't get it installed. Both on linux (debian) and MacOS

**Comment by onoff0** [comment]
in ubuntu there is a formatting problem, since the source was loaded on
windows and then put on githib. you have to modify the file .eslintrc.js
"linebreak-style": with windows

Il Ven 2 Nov 2018, 10:47 lorenzoromani1983 <notifications@github.com> ha
scritto:

**Comment by lorenzoromani1983** [comment]
Yes I acknowledged! I have been able to build and run on kali Linux, now I wonder if the sw is still fully customizable as open source....

**Comment by onoff0** [comment]
Il giorno ven 2 nov 2018 alle ore 22:15 lorenzoromani1983 <
notifications@github.com> ha scritto:

**Comment by onoff0** [comment]
Il giorno ven 2 nov 2018 alle ore 22:15 lorenzoromani1983 <
notifications@github.com> ha scritto:
